src/Dusk.php

```diff
@@ -136,6 +136,16 @@ public function noSandbox()
     {
         return $this->addArgument('--no-sandbox');
     }
+    
+    /**
+     * Disable the SSL protection.
+     *
+     * @return $this
+     */
+    public function noSsl()
```

Comment:
Can we change it to `ignoreSslErrors()`, `ignoreSslProtection()` or `noSslProtection()`. `noSsl()` seem too generic

Reply:
You're right. I just pushed the change on my repo 👍